easybuild/easyblocks/n/netcdf.py

```diff
@@ -71,6 +71,27 @@ def configure_step(self):
 
             CMakeMake.configure_step(self)
 
+        if LooseVersion(self.version) >= LooseVersion("4.3.3"):
```

Comment:
is this version check strictly needed? this probably applies to all netCDF versions where CMake is used to configure (i.e. 4.3.x and up)?